badCode.rb

```diff
@@ -0,0 +1,6 @@
+def badName
+  if something
```

Comment:
Style/GuardClause: Use a guard clause instead of wrapping the code inside a conditional expression.